Automattic/jetpack
# Add `<9.12.0` to `.engines.pnpm`

## Proposed changes:

We're waiting on updating to pnpm 9.12+ due to the regression of pnpm/pnpm#3935. Update `.engines.pnpm` so Renovate stops trying to use 9.12 and then failing the "Lock files are up to date" check.

### Other information:


## Jetpack product discussion


p1727989031867039-slack-CDLH4C1UZ

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Fork the repo then run renovate, see if it uses the right version.
  * Or see https://github.com/anomiex/jetpack/actions/runs/11505569400/job/32027557957#step:5:56891 where I did that already.
